pkg/proxy/proxy.go

```diff
@@ -517,6 +527,43 @@ func Init(cfg *Config) (*Proxy, error) {
 	return p, nil
 }
 
+// ExtractRequestData takes an interface with request data from a Stripe event payload
+// and properly parses it into a StripeRequest struct before returning it
+func ExtractRequestData(data interface{}) (StripeRequest, error) {
+	var req StripeRequest
+
+	reqDataValue := reflect.ValueOf(data)
+	// here we check the type of the RequestData as it could be either a string or a map
+	// this depends on which API version is in use when listening to events
```

Comment:
Should we specify which version? It's 2017-05-25 right?

Reply:
👍  great idea to document, and it's all versions including and before that I believe